[FIX] Alert styling og ikoner

KenAJoh opened the pull request:
**WIP**
Begynt synk med Figma


- [x] Oppdatert Warning og Error ikoner fra figma -> ikonpakke -> Brukt i Alert

KenAJoh committed:
oppdatert alert bakgrunn

KenAJoh committed:
lint

KenAJoh committed:
Merge branch 'master' into patch/alert

KenAJoh committed:
oppdaterte ikoner for alert

KenAJoh committed:
riktig bakgrunn for warning

andnorda commented on @navikt/core/css/alert.css:
```diff
@@ -104,12 +83,9 @@
-  );
+  box-shadow: inset 10px 10px 0 0 var(--navds-alert-color-warning-background),
+    inset 10px -4px 0 0 var(--navds-alert-color-warning-background),
+    inset 14px 0 0 0 var(--navds-alert-color-icon-background);
```
Før hadde utropstegnet i warning-ikonet samme farge som tekst for å få god nok kontrast med den orange bakgrunnen. Er det noe vi går bort fra?

KenAJoh replied:
Ja designet har nå hvit ! og mørkere ikon orange-500->orange-600 for å dekke kontrastkravet

andnorda replied:
Skjønner!

Jeg prøvde meg på en annen metode, men jeg vet ikke om denne ble så mye bedre
```suggestion
background: linear-gradient(white, white) center / 16% 60% no-repeat;
```

andnorda commented on @navikt/core/icons/svg/ErrorFilled.svg:
```diff
@@ -1,3 +1,3 @@
 <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M12 0C18.6274 0 24 5.37258 24 12C24 18.6274 18.6274 24 12 24C5.37258 24 0 18.6274 0 12C0 5.37258 5.37258 0 12 0ZM15.5714 7L17 8.42857L13.428 12L17 15.5714L15.5714 17L12 13.428L8.42857 17L7 15.5714L10.572 12L7 8.42857L8.42857 7L12 10.572L15.5714 7Z" fill="#262626"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12 24C18.6274 24 24 18.6274 24 12C24 5.37258 18.6274 0 12 0C5.37258 0 0 5.37258 0 12C0 18.6274 5.37258 24 12 24ZM16.5714 18L18 16.5714L7.42857 6L6 7.42857L16.5714 18Z" fill="#262626"/>
```
Jeg har aldri sett et sånt error-ikon før. Burde vi ikke bruke et uttrykk for å kommunisere feil som folk kjenner fra før?

KenAJoh replied:
Designerne har funnet frem til denne nå etter flere møter slik jeg har skjønt det. Men er med på å gi litt pushback da jeg heller ikke kjenner til dette utrykket for error.

andnorda commented on @navikt/core/css/alert.css:
```diff
@@ -3,49 +3,28 @@
-  --navds-alert-color-error-background: var(
-    --navds-semantic-color-feedback-danger-background
-  );
+  --navds-alert-color-error-background: var(--navds-global-color-red-50);
```
Burde denne endringen gjøres i den semantiske tokenen, så alle som bruker `danger`-fargene endres og ikke bare alert?

KenAJoh replied:
Ja bør kanskje det, kan høre med designer

KenAJoh committed:
[FIX] parentElement prop på Modal (#1450)

github-actions[bot] committed:
Lerna versionbump [ci skip]

 - @navikt/ds-react@0.17.1
 - @navikt/ds-react-internal@0.12.1

KenAJoh committed:
[FIX] onSubmit kjører onSearch callback i `Search` (#1452)

github-actions[bot] committed:
Lerna versionbump [ci skip]

 - @navikt/ds-react@0.17.2
 - @navikt/ds-react-internal@0.12.2

KenAJoh committed:
[FIX] Synket ConfirmationPanel med Figma (#1437)

Co-authored-by: Andreas Nordahl <andnorda@gmail.com>

github-actions[bot] committed:
Lerna versionbump [ci skip]

 - @navikt/ds-css@0.16.2

KenAJoh committed:
[FIX] Search: form -> div (#1454)

github-actions[bot] committed:
Lerna versionbump [ci skip]

 - @navikt/ds-react@0.17.3
 - @navikt/ds-react-internal@0.12.3

KenAJoh committed:
[FIX] Bedre width for search (#1455)

github-actions[bot] committed:
Lerna versionbump [ci skip]

 - @navikt/ds-css@0.16.3

andnorda committed:
[fix] Helptext uu fix (#1407)

* storybook no div in p fix

* move title inside svg

* extra helptext in story

* remove duplicate title

* remove aria-props and refactor

* remove unused style

* simplify styling some

* default help text content to body-short

* cleanup

* tabindex 0 på popover

* flyttet story

* fjernet tabindex

* Update @navikt/core/react/src/help-text/HelpText.tsx

Co-authored-by: Ken <26967723+KenAJoh@users.noreply.github.com>

Co-authored-by: Ken <ken.aleksander@gmail.com>
Co-authored-by: Ken <26967723+KenAJoh@users.noreply.github.com>

github-actions[bot] committed:
Lerna versionbump [ci skip]

 - @navikt/ds-css@0.16.4
 - @navikt/ds-react@0.17.4
 - @navikt/ds-react-internal@0.12.4

KenAJoh committed:
Update @navikt/core/css/alert.css

Co-authored-by: Andreas Nordahl <andnorda@gmail.com>

KenAJoh committed:
fikset ikoner

KenAJoh committed:
organized imports

KenAJoh committed:
oppdatert fargebruk css

KenAJoh committed:
Merge branch 'master' into patch/alert
